src/django/api/reports/percent_data_by_source_type.sql

```diff
@@ -21,9 +23,8 @@ JOIN (
         FROM api_facilitymatch m
             JOIN api_facilitylistitem i on m.facility_list_item_id = i.id
             JOIN api_source s on i.source_id = s.id
-            JOIN api_contributor c ON s.contributor_id = c.id
         WHERE m.status NOT IN ('REJECTED', 'PENDING')
-        GROUP BY m.facility_id, c.id
+        GROUP BY m.facility_id, s.contributor_id, s.source_type
```

Comment:
Adding the `source_type` to the grouping here prevents the filtering out of rows where the contributor has resubmitted a facility using a different source type.